ts/output/common/Wrapper.ts

```diff
@@ -741,14 +741,16 @@ export class CommonWrapper<
    */
   protected copySkewIC(bbox: BBox) {
     const first = this.childNodes[0];
-    if (first?.bbox.sk) {
-      bbox.sk = first.bbox.sk;
-    }
-    if (first?.bbox.dx) {
-      bbox.dx = first.bbox.dx;
+    if (first) {
+      if (first.bbox.sk) {
```

Comment:
Will `bbox` exist here, since you explicitly check for it later in line !753?

I don't think we gain much with the `if` in line 744. It would  look considerably cleaner otherwise.

Reply:
I guess line 744 should be `if (first?.bbox) {` so that you don't need the question mark check repeated in the other two.  But I can change it to doing that twice, as `first?.bbox?.sk` and `first?.bbox?.dx` if you prefer.